packages/medusa/src/api-v2/admin/products/route.ts

```diff
@@ -54,11 +54,9 @@ export const GET = async (
     entryPoint: "product",
     variables: {
       filters: filterableFields,
-      order: req.listConfig.order,
-      skip: req.listConfig.skip,
-      take: req.listConfig.take,
+      ...req.remoteQueryConfig.variables,
```

Comment:
Reading this again, `variables` is quite ambiguous I'd say, can we maybe make it more specific, eg. `pagination`?

Reply:
yes we could, it includes, skip, take but also ordering, I ve named it variables to match the object API and recognize it. also, I am fine with pagination but order would be out of context in that case no?

Reply:
@adrien2p well ordering is related to pagination IMO, so it would be fine as such. Anyway that's my opinion, happy to hear what others think, but variables is too generic

Reply:
I am fine with pagination, I ve renamed and updated the types